targets/mbedos5/source/jerry_port_mbed.c

```diff
@@ -90,10 +90,36 @@ jerry_port_get_time_zone (jerry_time_zone_t *tz_p) /**< timezone pointer */
 /**
  * Implementation of jerry_port_get_current_time.
  *
- * @return current timer's counter value in microseconds
+ * @return current timer's counter value in milliseconds
  */
 double
 jerry_port_get_current_time ()
 {
-  return (double) us_ticker_read ();
+  static uint32_t last_tick = 0;
+  static time_t last_time = 0;
+  static uint32_t skew = 0;
+
+  uint32_t curr_tick = us_ticker_read (); /* The value is in microseconds. */
+  time_t curr_time = time(NULL); /*  The value is in seconds. */
+  double result = curr_time * 1000;
+
+  /* The us_ticker_read () has an overflow for each UINT_MAX microseconds
+   * (~71 mins). For each overflow event the ticker-based clock is about 33
+   * milliseconds fast. Without a timer thread the milliseconds part of the
+   * time can be corrected if the difference of two get_current_time calls
+   * are within the mentioned 71 mins (note: because the comparison of two
+   * uint32 numbers, this interval has to be divided by 2). Above that
+   * interval we can assume that the milliseconds part of the time is
+   * negligibe.
+   */
+  if (curr_time - last_time > 1800) {
```

Comment:
Well, it should 2147 sec (~35 mins) if we are about to be as correct as possible. Probably the comment does not say anything about the 1800 seconds (30 mins) interval, but it could be a more user friendly threshold than the 2147. It is easier to accept the 30 mins limit than the 35 mins one in order no to rely on milliseconds based timers above that interval.
Do you have any other option?